### ISSUE TITLE
object detection with resnet-34

### ISSUE THREAD

**Issue by paul8666** [amrit110/masterThesis#2]
![bbbb](https://cloud.githubusercontent.com/assets/22220669/18747376/4866c406-80cd-11e6-96e5-0fdef4741f67.png)
I tried your function with some chages:
modified main.lua adding a inference section:
I used the default resnet-34.t7

dofile 'sequenceInference.lua'
--inference
setParameterNetwork('model')
inferSequence()--

the input image is correct ( I saved the image ) but when I check the number of detection for each 3 classes ( selected:numel() ) is equal to zero. I attached an example of heatmap that I obtained
do you have any idea?

**Comment by amrit110** [comment]
The resnet is a pretrained net on the IMAGENET dataset. We use the pretrained network and add our own part to it for classification and bounding box regression. But one would have to train to fine-tune this model on the KITTI dataset to obtain results. We only have released the code in this repository, not the fully trained model. So, you have to train first and then try inference. Training for fine tuning roughly takes 6 hours on an Nvidia titanx.

**Comment by paul8666** [comment]
Ok, I fine tuned the resnet-34.t7 on kitti object detection dataset, The heatmaps looks better than the previous one, but I think it does not look similar to you test, I need to change some parameter for the training?

![aaaa](https://cloud.githubusercontent.com/assets/22220669/18777667/ab2f63ce-8170-11e6-927a-c2e82e2c1e81.png)
![bbbb](https://cloud.githubusercontent.com/assets/22220669/18777668/ab304780-8170-11e6-8d5e-21b2f9dd5e5b.png)

**Comment by amrit110** [comment]
not sure how you did the fine tuning. ideally, you should freeze the pretrained network and only train the parts we added first, and you typically want to do this for enough number of minibatches. then u backprop to the entire network with a lower learning rate. the training code in the main.lua file is just for direction, not necessarily what we used. You might have to play around a bit to get a sense of what hyper parameters to use.
